README.md

```diff
@@ -17,14 +17,25 @@ The Terra Azure Relay Listener establishes a bi-directional channel with Azure R
 
 `relayConnectionName`: Hybrid Connection name. Must the same value as the EntityPath.
 
-`listener.targetProperties.targetHost`: The local or private endpoint where the listener must forward all requests.
+`listener.targetProperties.targetHost`: The default local or private endpoint where the listener must forward all requests.
 
 `requestInspectors`: A list of request inspectors to be enabled.
 
 `listener.targetProperties.removeEntityPathFromHttpUrl` If `true` the HTTP request to the target won't include the Entity Path (Hybrid Connection name) in the URL. The default value is `false`.
 
 `listener.corsSupportProperties.preflightMethods` Methods that we support CORS. Default to `OPTIONS, POST, PUT, GET, DELETE, HEAD, PATCH`.
 
+`listener.targetProperties.targetRoutingRules` A list of routing rules. A rule is a tuple of the string the URI must contain for a match and the target host.
+The default `targetHost` is used. Example configuration:
+
+```yaml
+  targetProperties:
+    targetHost: "http://localhost:8080"
+    targetRoutingRules:
+      -
+        pathContains: "welder"
+        targetHost: "http://localhsot:8081"
```

Comment:
how will `listener.targetProperties.targetRoutingRules` look like in env? is it going to be sth like `LISTENER_TARGETPROPERTIES_TARGETROUTINGRULES_0_PATHCONTAINS = "welder"`? anyway we can make it sth that doesn't involve `_<number>`?